packages/components/Molecules/Search/SearchResultsLayout.vue

```diff
@@ -27,7 +27,7 @@
         </slot>
       </rpl-row>
       <div v-if="searchResults.length === 0 && !loading" class="rpl-search-results-layout__no-results-msg">
-        <slot name="noresults"><p>Sorry! We couldn't find any matches</p></slot>
+        <slot name="noresults"><p> {{ noResultsMsg }} </p></slot>
```

Comment:
Can we please remove trailing spaces? This should be fine as browser will auto remove it but saves two bits (: